src/server/enrollment-system.js

```diff
@@ -63,22 +63,163 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
-function zeroPadNumber(number, padding) {
-  return (new Array(padding + 1).join('0') + number).slice(-padding);
+/**
+ * Returns a string left-padded with '0' of the given width.
+ *
+ * Warning: If 'number' is wider than 'width' characters, only the right-most 'width'
+ * characters are returned.
+ *
+ * TODO: should this explicitly check if 'number' and 'width' are actually numbers?
+ *
+ * @param {number} number The number you want left-padded with '0' characters.
+ * @param {number} width The width of the resulting string.
+ * @returns {String}
+ *
+ * @example
+ * // returns "0001"
+ * zeroPadNumber(1, 4);
+ * @example
+ * // returns "15"
+ * zeroPadNumber(31415, 2);
+ */
+function zeroPadNumber(number, width) {
+  return (new Array(width + 1).join('0') + number).slice(-width);
 }
 
+/**
+ * Returns true when the provided year is a leap-year in the Gregorian calendar.
+ *
+ * @param {number} year The year you want to test.
+ * @returns {boolean}
+ *
+ * @example
+ * // returns true
+ * isLeapYear(2012)
+ *
+ * @example
+ * // returns false
+ * isLeapYear(2011)
+ *
+ * @example
+ * // returns [ true, false, false, false, true, true, true ]
+ * [1600,1700,1899,1900,2000,2016,2012].map(isLeapYear)
+ */
+function isLeapYear(year) {
```

Comment:
Can you add a TODO in here and file an issue against me to double check the date logic?  This stuff always scares me.